vme/src/db.cpp

```diff
@@ -486,27 +433,16 @@ void generate_zone_indexes()
 
         slog(LOG_ALL, 0, "Indexing %s AC[%3d] LL[%d] PO[%d]", filename, access, loadlevel, payonly);
 
-        z = new (zone_type);
+        fstrcpy(&cBuf, f);
+        z = new zone_type{(char *)cBuf.GetData(), zone};
         g_zone_info.no_of_zones++;
```

Comment:
Wonder if this global value shouldn't be incremented in the zone_type contructor.

Reply:
I get rid of it in the next PR - the map knows how many elements are in it, no point maintaining it twice